### DISCUSSION TITLE
Gathering your WHITELIST and BLACKLIST applications

### DISCUSSION THREAD

**Discussion by nefares** [nefares/Background-Muter#28] [Ideas]
Hello everyone,

With the goal to gather an exhaustive "default" application list for Background-Muter, I would like you to share your lists here so that we can congregate them all into a master list to be used by all users. 

To clarify:
- **WHITELIST (default):** This is the only mode currently supported. It means that all applications in this list are allowed to stream audio in the background while everything else is muted.
- **BLACKLIST:** This may be added in the future. It means that all applications in this list are MUTED while in the background, while everything else is allowed.

### Please post your current (or wished) list in the following format:

```
Type: <WHITELIST> or <BLACKLIST>
Application List: <comma separated process list> 
```

Note that you can directly copy your current whitelist from  Background-Muter by going to the "Control" tab and copying everything under "Mute Exceptions".

### Example (the currently default list):

```
Type: WHITELIST
Application List: chrome,vlc,firefox,WWAHost,,mpc-hc64,msedge
```

Thanks!

**Comment by Semidio** [comment]
Type: WHITELIST
Application List:mpv,msedge,PotPlayerMini64,AMPLibraryAgent

**Comment by lamrongol** [comment]
Type: WHITELIST
Application List: vivaldi,MusicBee

**Reply by lamrongol** [reply]
(Question) Is it sufficient only process names? I think it is difficult to determine addding apps to "default" list without knowing app info. For example, "vivladi" -> browser, "MusicBee" -> media player. And basic info, app name and official site.

**Comment by NonMuss** [comment]
Type: WHITELIST
Application List: chrome,mpc-hc64,vlc,firefox,WWAHost,msedge,mpv,PotPlayerMini64,AMPLibraryAgent,brave

**Comment by 22rw** [comment]
Type: WHITELIST
Application List: mpc-hc64,vlc,firefox,WWAHost,mpv,Discord,Spotify,WhatsApp,Hourglass,Signal,Video.UI

**Comment by RobbyMeeker** [comment]
Very much would like a blacklist feature. I do not want to manually whitelist all of my other applications.

**Comment by Ansem93** [comment]
Type: WHITELIST
Application List: Amazon Music, Spotify, Discord, WhatsApp, Brave, Telegram, wallpaper64

**Comment by HisFanciness** [comment]
Type: BLACKLIST
Application List: Chrome, VLC, Firefox, MSEdge, Totoro, Miru, YouTube Music, Discord, Telegram, WhatsApp, Steam, Stremio, Hayase, Popcorn-Time, Nvidia Broadcast

Many more, but mostly games and I can't list tens of games here.
